nautobot/dcim/migrations/0053_create_hardware_family_model.py

```diff
@@ -0,0 +1,60 @@
+# Generated by Django 3.2.23 on 2024-01-03 04:47
+
+import uuid
+
+import django.core.serializers.json
+from django.db import migrations, models
+import django.db.models.deletion
+
+import nautobot.core.models.fields
+import nautobot.extras.models.mixins
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("extras", "0102_set_null_objectchange_contenttype"),
```

Comment:
I think this should depend on extras `0001_initial_part_1 ` since that's where the `Tag` and `TaggedItem` models are created and those are your only ForeignKeys to extras.